### PR TITLE
URL shorten-er project

### PR DESCRIPTION
**Pull request by alifa98**

### CODE REVIEW

File: Projects/3-Advanced/URL-Shortener.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## User Stories
+
+-   [ ] User can enter a URL and get a short code.
+-   [ ] User can enter a short code format and get original URL.
+
+
+## Bonus features
+
+-   [ ] User can see the stats of shortened URL (like number of visits and so on)
+- [ ] User can provide a prefered code name to URL (You must check the code is exists or not)
```
Comment (sergeyklay) [inline Projects/3-Advanced/URL-Shortener.md:21]:
```suggestion
-   [ ] User can provide a prefered code name to URL (You must check the code is exists or not)
```

File: README.md [hunk 2]
```diff
@@ -141,6 +141,7 @@ required to complete them.
 | [Slack Archiver](./Projects/3-Advanced/Slack-Archiver.md)                     | Archive Slack Messages                                              | 3-Advanced |
 | [Spell-It App](./Projects/3-Advanced/SpellIt-App.md)                          | A twist on the classic Speak N Spell game                           | 3-Advanced |
 | [Survey App](./Projects/3-Advanced/Survey-App.md)                             | Define, conduct, and view a survey                                  | 3-Advanced |
+| [URL Shortener](./Projects/3-Advanced/URL-Shortener.md)                       | Shorten URls                                                        | 3-Advanced |
```
Comment (sergeyklay) [inline README.md:144]:
```suggestion
| [URL Shortener](./Projects/3-Advanced/URL-Shortener.md)                       | Shorten URLs                                                        | 3-Advanced |
```

**Comment by alifa98** [comment]
Hi,
So, should I delete the fork? or do you want to merge it?

thank you.

**Comment by abderrahmen-bmz** [comment]
Delete it for me

**Comment by aldairandre** [comment]
# Hello everyone 👋

I hope this message finds you well. As you may know, the original repository for app ideas has not been updated in a while and is no longer actively maintained. However, I'm excited to let you know that a new repository has been created to continue this project (https://github.com/vincent0426/app-ideas).

The new repository aims to provide a comprehensive list of practical and achievable app ideas that developers of all levels can work on to improve their skills and build their portfolios. We welcome contributions from anyone who wants to add their own app ideas or improve the existing ones, and we are also looking for volunteers who can help maintain the repository and keep it up to date.

If you are interested in contributing or maintaining this project, please head over to the new repository (https://github.com/vincent0426/app-ideas) ⭐ and take a look at the existing issues and pull requests. Feel free to open a new issue if you have an app idea to suggest or if you have any questions about the project.

Thank you for your attention, and I hope to see you in the new repository soon! 💖

<br/>

⚠️ This message was generated by a robot 🤖, if this message reached you, consider leaving a star for this robot, repository link

⭐ https://github.com/aldairandre/github-bot ⭐

**Comment by alifa98** [comment]
The EnD

**Pull request closed without merging**